tools/eosiocpp.in

```diff
@@ -5,7 +5,27 @@ if [ "${EOSIO_BIN_INSTALL_DIR}" == "." ]; then
   EOSIO_BIN_INSTALL_DIR=`pwd`
 fi
 EOSIO_INSTALL_DIR=`dirname ${EOSIO_BIN_INSTALL_DIR}`
-ABIGEN=${EOSIO_INSTALL_DIR}/bin/eosio-abigen
+if [ -x "@CMAKE_BINARY_DIR@/programs/eosio-abigen/eosio-abigen" ]; then
```

Comment:
I worry about packaging and installing a script that has the full CMAKE_BINARY_DIR in the resulting file.  Won't that leak some information that could be considered sensitive about the system that built the binary package (like user name of the builder etc).  

Can we do something where it discovers all of these paths relative to the directory where the script resides so that we don't have to hard code the absolute path?  I realize this makes `eosiocpp` fragile when you copy/move it around but, that seems like a better trade?

Reply:
It could leak information depending on how it's built.  It does not have to be sensitive information, especially on dedicated build boxes.  At present, we're not distributing any package format except Docker, which I don't think carries ownership or a home directory path within it.  Finally, this is a stopgap until the wasmsdk repo is ready for prime time.

Reply:
yeah, docker should do a decent job of sanitizing paths... I'm checking right now and will release my objection if it looks benign.